services/drone/drone-build.tester.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+const Joi = require('joi')
+const { isBuildStatus } = require('../build-status')
+const t = (module.exports = require('../tester').createServiceTester())
+const { mockDroneCreds, token, restore } = require('./drone-test-helpers')
+
+// Drone build (cloud)
+
+t.create('cloud build status on default branch')
+  .get('/drone/drone.json')
+  .expectBadge({
+    label: 'build',
+    color: 'brightgreen',
```

Comment:
I'd suggest removing the color checks from the expect statements here, as the colors are associated to a specific value that will (inevitably) fluctuate.

Reply:
Done!